src/test/desmoHub-test.ts

```diff
@@ -48,6 +48,13 @@ describe('DesmoHub Tests', function () {
     });
   });
 
+  describe('Retrieve TDD list', function () {
+    it('should retrieve a list of tdds', async () => {
+      const tddList = await desmohub.getTDDList(1, 3);
+      expect(tddList.length).to.equal(3);
+    });
+  });
+

```

Comment:
Need also failing tests for an empty response, zero length, start greater than the end. Start greater than TDD length.

Reply:
I cannot test the "empty TDDStorage" case as it would require to delete the registered TDDs in the deployed contract, which is not possible since the contract doesn't allow for TDD deletions.

I will implement all the other suggested cases.

Reply:
Ok, then it is an issue of SDK tests. The unit tests should not rely on the deployed contract but they should use a mock-up. Perhaps we should introduce hardhat as a dev dependency and write SDK tests with that.